skimage/measure/__init__.py

```diff
@@ -1,3 +1,5 @@
+"""Measurement of image properties, e.g., region properties and contours."""
```

Comment:
```suggestion
"""Measurement of image properties, e.g., region properties, contours."""
```
(just to keep the same style as previously)